docusaurus/docs/iOS/basics/logs.md

```diff
@@ -31,8 +31,7 @@ By default, the logs will provide basic text to your console. Still, in the SDK,
 
 ```swift
 LogConfig.formatters = [
-    PrefixLogFormatter(prefixes: [.info: "ℹ️", .debug: "🛠", .warning: "⚠️", .error: "🚨"]),
-    PingPongEmojiFormatter()
+    PrefixLogFormatter(prefixes: [.info: "ℹ️", .debug: "🛠", .warning: "⚠️", .error: "🚨"])
 ]
 ```
```

Comment:
We do not have an example of `PingPongEmojiFormatter` class. I think it's better to remove this line or copy-paste a code example from `StreamChatSample` target.